timescaledb/tutorials/grafana/visualizations/candlestick.md

```diff
@@ -0,0 +1,167 @@
+
+<procedure>
+
+  With the use of the $bucket_interval variable, we are able to switch intervals. For example, switching to a 15-minute bucket interval gives you this data. 
+
+  1.  Switch your bucket interval to 15-min from the dropdown
+      
+      <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/candlestick/timebucket_dropdown.png" alt="Screenshot of the Grafana variable dropdown."/>
+
+  1.  Refresh the dashboard to get the updated chart
+      
+      <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/candlestick/15_min.png" alt="Screenshot of the Grafana variable dropdown."/>
+
+  These queries execution took about 6+ seconds, to decrease query execution time to sub-seconds it's suggested to use the continuous aggregates, which we will cover in a different section.
+
+</procedure>
+
+## Show transaction volumes in a candlestick plot
+
+In addition to looking at the price changes for each stock, you can look at its traded volumes.
+This shows you how much the stock is being traded during the bucket interval. 
+
+The `stock_real_time` hypertable contains a column with the daily cumulative traded volume. You can use this to calculate the volume of data for each bucket. 
+
+First, find the maximum `day_volume` value for a symbol within a bucket.
+Then, subtract each maximum from the previous bucket's maximum. The
+difference gives the traded volume for that bucket.
+
+<procedure>
+
+### Showing transaction volumes in a candlestick plot
+
+1.  Create a new candlestick panel with the following query:
+    ```sql
+    SELECT
+      time_bucket('$bucket_interval', time) AS time,
+      symbol,
+      FIRST(price, time) AS "open",
+      MAX(price) AS high,
+      MIN(price) AS low,
+      LAST(price, time) AS "close",
+      MAX(day_volume) - LAG(max(day_volume), 1) OVER(
+        PARTITION BY symbol
+        ORDER BY time_bucket('$bucket_interval', time)
+      ) AS bucket_volume
+    FROM stocks_real_time
+    WHERE symbol =  $symbol
+      AND time > $__timeFrom()::timestamptz and time < $__timeTo()::timestamptz
+    GROUP BY time_bucket('$bucket_interval', time), symbol;
+    ```
+
+1.  Refresh the dashboard to get the updated chart.
+    
+     <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/candlestick/volume_Distribution.png" alt="Screenshot of Grafana candlestick showing the stock volume distribution and price for $AMZN."/>
+
+    At the bottom of the plot, you see the trade volume for each time bucket.
+
+</procedure>
+
+Candlestick charts are a great way to visualize financial data. This tutorial shows you how to u
+se TimescaleDB to generate candlestick values (open, high, low, close) from raw data in a
+hypertable. It also shows you how to query the traded volume for each time interval. 
```

Comment:
This feels like it should be at the top, or maybe needs an "in conclusion"

Reply:
yea, just added it.